WinRT.Runtime/Projections/IDictionary.net5.cs

```diff
@@ -14,12 +14,13 @@
 
 namespace Windows.Foundation.Collections
 {
+    [global::WinRT.WindowsRuntimeType]
     [Guid("3C2925FE-8519-45C1-AA79-197B6718C1C1")]
     interface IMap<K, V> : IIterable<IKeyValuePair<K, V>>
     {
         V Lookup(K key);
         bool HasKey(K key);
-        IMapView<K, V> GetView();
+        IReadOnlyDictionary<K, V> GetView();
```

Comment:
the pattern in the PR seems to be to return nested projected types from WinRT interfaces:
IIterable->IEnumerator
IMap->IReadOnlyDictionary

what's the reason for that?

Reply:
As mentioned in the description, this is required to enable the managed code to return a reference of the managed object to the native side instead of an instance of `ToAbiHelper` which indirectly points to the managed object.